src/redux/news/news.sagas.js

```diff
@@ -5,7 +5,7 @@ import { setError } from '../error/error.actions';
 import getItems from '../../utils/client';
 import { GET_NEWS, GET_NEWS_ARTICLE } from './news.types';
 
-function* handleNewsLoad() {
+export function* handleNewsLoad() {
```

Comment:
Why have you added  `export` here? This saga function should be used only inside this file and root news saga

Reply:
I exported it for tests